### ISSUE TITLE
Remove Header Padding

### ISSUE THREAD

**Issue by julianrinaldi** [thomasloven/lovelace-fold-entity-row#121]
It would be nice to be able to remove the padding on the right side of the header. It doesn't add anything visually or functionally. In fact, I think it hurts functionality by reducing the the amount of text that can be shown if you have an entity with a longer name.

<img width="659" alt="Screen Shot 2020-08-22 at 9 40 29 AM" src="https://user-images.githubusercontent.com/492435/90957497-e6967b80-e45b-11ea-945d-9486e5ba261b.png">

**Comment by noxhirsch** [comment]
Here's a little workaround with card-mod: 
```
head:
  entity: group.lights
items:
  - light.one
  - light.two
  - light.three
type: 'custom:fold-entity-row'
style: 'ha-icon{position:absolute;right:-15px!important;}'
```

**Comment by julianrinaldi** [comment]
Thanks, that moves the icon over, but leaves the toggle where it was. Do you know the css to move the toggle over as well?

**Comment by noxhirsch** [comment]
That's correct - it only moves the arrow icon. The toggle is two shadow-root layers deeper, so it's not as easy.
For me the arrow icon on the correct position was enough.
You can try with that guide and e.g. chrome dev tools: https://github.com/thomasloven/lovelace-card-mod#advanced-usage

**Comment by marcelodavanzo** [comment]
For some reason the toggle icon has a width of 40px which is too wide and makes it look indented from the right. Using card-mod the snippet below will set it to 24px which gets rid of the extra empty space and positions it correctly:

```
type: 'custom:fold-entity-row'
style:
  .: |
    div#head {
      --toggle-icon-width: 24px;
    }
```

It's a stop-gap solution though. I'd much rather this got fixed directly in the add-on.

**Comment by thomasloven** [comment]
This has been fixed in 20.0.0
